### DISCUSSION TITLE
28

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#770] [Comments]
http://jojoldu.tistory.com/28

**Comment by jojoldu** [comment]
작성자: 박수영

글 잘읽었어요 감사합니다!!ㅎㅎ

**Comment by jojoldu** [comment]
작성자: 창천향로

와! 수영님 방문해주셔서 감사합니다^^
잘 읽어주셔서 뿌듯하네요!!
앞으로도 자주 방문 부탁드리겠습니다^^

**Comment by jojoldu** [comment]
작성자: 지나가는이

글 잘 읽을게요 감사합니다!!

**Comment by EastStar1129** [comment]
잘읽었습니다!! 혹시 첫번째 그림에서 web server안에 servlet container그림을 넣으신 이유를 여쭈어봐도 될까요?? web server와 servlet container를  각각 독립적인 서비스로 이해하고있었는데 그림에는 포함되어 있어 질문드립니다!

**Comment by taekwon-dev** [comment]
@EastStar1129

확실하지는 않지만, 제 생각에는 일반적으로 Nginx (프록시) 서버를 통해서 Spring Application을 서비스 하는 형태가 많아서 해당 그림으로 표현하시지 않았을까 생각해요!
